reto.exs

```diff
@@ -0,0 +1,23 @@
+defmodule Recursion do
+  def print_multiple_times(msg, cantidad, cont, limit_inf, limit_sup) when cont <= limit_sup do
+    msg = msg <> "#{cont}"
+    numero = String.to_integer(msg)
+    if cont >= limit_inf do
+      IO.puts(numero)
+      cond do
+      rem(numero,3) == 0 -> print_multiple_times(msg, cantidad + 1, cont + 1, limit_inf, limit_sup)
```

Comment:
If you see `print_multiple_times` is being called in both cases what if we redefine a new amount (cantidad) on a variable and only sum 1 or 0 if the number is divisible by 3?

```suggestion
      print_multiple_times(msg, cantidad + (rem(numero,3) == 0 && 1 || 0) , cont + 1, limit_inf, limit_sup)
```

With something like that e.g, you will be avoiding a case statement!

PD: you can also move the `cantidad + rem(numero,3) == 0 && 1 || 0` logic to another function like sum_if_divisible

Reply:
Also, consider avoiding magic numbers! you can move `3` to a constant or receive it as a param, this would be a good practice ⭐